### ISSUE TITLE
Is this tweak is working on iOS14.3 ?

### ISSUE THREAD

**Issue by liliass** [brendonjkding/VolumeMixer#5]
I always use wechat video call and watch YouTube in the same time. By default, I can’t adjust the volume in YouTube. Not sure if the tweak works for this ? 

Thanks for your effort.

**Comment by brendonjkding** [comment]
It's a known issue. Doesn't work on iPad, and may work on iPhone.

**Comment by badger200** [comment]
@brendonjkding Any plans for iPad support? I’ve been running it for months and I’m not entirely certain but I think it’s functioning to allow me to set certain apps to mix sound (and not interrupt whatever I’m playing in the background). 

It’s not entirely clear what the settings do, Web Audio and the one below that in particular.

**Comment by brendonjkding** [comment]
Perhaps I was not clear, so let me clarify.

This Tweaks works on iOS 14.3 (and iPadOS 14.3). 
If any function not works, it's not an iOS version-related issue.

Currently It's not possible to adjust the volume in Youtube on iPad using this tweak. 
Because videos in Youtube app are actually on a webpage.
Some videos on webpage can be controlled by normal APIs but others can't. 
Unfortunately Youtube is the latter.
Maybe I missed some APIs. 

I don't think I'm able to fix it at the moment.
I will look into this after I learn more about web stuff.

As for the former, some users report webpage crash. So heres comes the Web Audio option (turn off all hooks) and the one below that(only turn off c-level hook).

**Comment by badger200** [comment]
I’m running iOS 12.4 on my iPad and I have never been able to get the VolumeMixer display to function. It pops up an empty blurred window, with no controls displaying, and then typically causes a SpringBoard crash. So I just avoid ever calling the mixer display (disappointing yes but the mixing functions are still very useful!)

Have you ever seen it function properly on an iPad? Which iOS version(s)?

On my iPhone 6S iOS 13.3 the mixer panel displays no problem.

BTW even on my iPhone this tweak’s mixer panel seems very prone to causing springboard crashes. It was such a problem I just disabled the functions to call the mixer panel on both my devices. I don’t know if it’s conflicting with some other tweak I have installed or what?

Is anybody using this tweak successfully with 100% stability? What iOS?

I’m on unc0ver 4.3.1, 5.3.1, and now 6.1.1 with Substitute 2.0.9 but prior versions all exhibited the same problems.
